### PR TITLE
[feat] Split code in the single run page

### PR DESCRIPTION
**Pull request by arsengit**
- Used React `Lazy loading` to avoid large size chunk in single run page
- Fixed `RunOverViewSidebar` width
- Tuned `Archive/Unarchive` button styles in `settings tab`
- Added `Spinner` component
- Fixed `Card` component padding

### CODE REVIEW

**Comment by arsengit** [comment]
Resolve #1484

File: aim/web/ui/src/pages/RunDetail/RunOverviewTab/components/Packages/RunOverviewTabPackagesCard.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export interface IRunOverviewTabPackagesCardProps {
-  packages:  null | { [key: string]: string };
+  packages: null | { [key: string]: string };
```
Comment (KaroMourad) [inline aim/web/ui/src/pages/RunDetail/RunOverviewTab/components/Packages/RunOverviewTabPackagesCard.d.ts:2]: i think here `packages`  can't be `null` already

Comment (KaroMourad) [reply aim/web/ui/src/pages/RunDetail/RunOverviewTab/components/Packages/RunOverviewTabPackagesCard.d.ts:2]: similar cases we have with `envVars` and `cliArgs`

File: aim/web/ui/src/pages/RunDetail/RunOverviewTab/RunOverviewTab.tsx [hunk 2]
```diff
@@ -34,55 +34,85 @@ function RunOverviewTab({ runData, runHash }: IRunOverviewTabProps) {
...
+    if (!_.isEmpty(runData?.runParams)) {
+      data.runParams = runData.runParams;
+    }
+    if (!_.isEmpty(runData?.runMetricsBatch)) {
+      data.runMetricsBatch = runData.runMetricsBatch;
+    }
+    if (!_.isEmpty(runData?.runSystemBatch)) {
+      data.runSystemBatch = runData.runSystemBatch;
+    }
+    if (!_.isEmpty(systemParams.arguments)) {
```
[the remark this answers is not in the document]
Comment (KaroMourad) [reply aim/web/ui/src/pages/RunDetail/RunOverviewTab/RunOverviewTab.tsx:49]: sadsd

**Review by rubenaprikyan — APPROVED** [review]
looks good

**Pull request merged**